packages/net/tcp_connection.pony

```diff
@@ -463,15 +463,22 @@ actor TCPConnection
     @pony_os_peername[Bool](_fd, ip)
     ip
 
-  fun ref expect(qty: USize = 0) =>
+  fun ref expect(qty: USize = 0) ? =>
     """
     A `received` call on the notifier must contain exactly `qty` bytes. If
     `qty` is zero, the call can contain any amount of data. This has no effect
     if called in the `sent` notifier callback.
+
+    `qty` can not exceed the max buffer size as indicated by the
```

Comment:
This docstring change is useful, but it doesn't explicitly say what happens if qty *does* exceed the max buffer size - it just says that it "can not" but doesn't state what happens in that case. So I suggest this minor change, or something similar:
```suggestion
    Errors if `qty` exceeds the max buffer size indicated by the
```

Reply:
Sounds good